mock-cmd/describe_connector.go

```diff
@@ -0,0 +1,51 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: cmd/meroxa/root/connectors/describe_connector.go
+
+// Package mock_connectors is a generated GoMock package.
+package mock_cmd
```

Comment:
I'm wondering if it would make more sense to have mocks in the folder (or subfolder) where they are needed. So this specific one could live for example in `cmd/cli/root/connector/mock`. Another option is to use the package `cmd/cli/root/connector/internal`, then it's clear that the mocks are meant to be used only inside the `connector` package, the compiler won't let you use it from other packages. Think about it.

Reply:
@lovromazgon most definitely! as I was migrating this, I thought it'd be nice to structure these. I'm not sure what's the best rout so I'll end up opening a PR just for that change once I'm done with all commands.